package.json

```diff
@@ -41,23 +41,23 @@
   ],
   "license": "MIT AND Apache-2.0",
   "devDependencies": {
-    "@tableland/local": "^0.0.5",
+    "@tableland/local": "^1.0.0-pre.3",
     "@types/mocha": "^10.0.1",
-    "@types/node": "^18.11.18",
-    "@typescript-eslint/eslint-plugin": "^5.48.2",
-    "@typescript-eslint/parser": "^5.49.0",
+    "@types/node": "^18.15.11",
+    "@typescript-eslint/eslint-plugin": "^5.58.0",
+    "@typescript-eslint/parser": "^5.58.0",
     "c8": "^7.12.0",
     "eslint": "^8.33.0",
     "eslint-config-prettier": "^8.6.0",
     "eslint-config-standard": "^17.0.0",
-    "eslint-config-standard-with-typescript": "^33.0.0",
+    "eslint-config-standard-with-typescript": "^34.0.1",
     "eslint-plugin-import": "^2.27.5",
     "mocha": "^10.2.0",
     "prettier": "^2.8.3",
     "ts-node": "^10.9.1",
-    "typescript": "^4.9.4"
+    "typescript": "^5.0.4"
   },
   "dependencies": {
-    "@tableland/sdk": "^3.1.7"
+    "@tableland/sdk": "^4.0.0-pre.7"
```

Comment:
Could we update this to 4.1.0+?

Reply:
Definitely!  Not sure how I missed that.